app/views/admin/media/_modal_content.html.slim

```diff
@@ -4,6 +4,7 @@ div.modal-header
   h4.modal-title = t("media.#{@media.new_record? ? 'add' : 'edit'}")
 
 div.modal-body.media-modal
+
```

Comment:
Not sure why this got added. Please remove as this PR has nothing to do with media.